harvester/src/main/java/com/github/reflectoring/infiniboard/harvester/source/url/UrlSourceJob.java

```diff
@@ -56,11 +63,42 @@ protected void executeInternal(ApplicationContext context, JobKey jobKey, Map co
 
             upsertResults(context, jobKey, results);
 
+        } catch (SSLHandshakeException e) {
+            String message =
+                    MessageFormat.format("Could not establish SSL connection to '{0}'. '{1}' is set to '{2}'.", url,
+                                         PARAM_DISABLE_SSL_VERIFY,
+                                         disableSslVerify);
+            if (disableSslVerify) {
+                LOG.warn(message);
+            } else {
+                LOG.error(message);
+            }
+        } catch (HttpHostConnectException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1166)